Augk mar25 editproducts

augustine24 opened the pull request:
Today I added just a bit of detail, I haven't gotten the add to cart button going, but I added a order quantity button that I will change later, and added a name variable. I also added a little bit of text at the top of the cart page so users know they're on the cart page like they do in the products page. 

Edit: Here's how it looks with the changes

![Screenshot 2024-03-26 142704](https://github.com/hinchley2018/learning-ecommerce/assets/106200215/54ff6611-4f8d-4ac5-a51e-93bd82c53644)

augustine24 committed:
changing product page

Added variables: items have a name variable and quantity can be edited and is displayed on the front end, also changed its size. Added a little message to tell users they're looking at their products aswell.

augustine24 committed:
package.json updated(?)

as title indicates there seems to have been an update or prettier package was added

augustine24 committed:
minor change products.js

changed stock verbiage to order quantity

sarmadH97 commented on frontend/package.json:
Already present in the file, please remove this duplication.

augustine24 replied:
Removed the duplication I'll make the PR request again with the changes

sarmadH97 commented:
@augustine24  Please attach the screenshot of the UI changes to the PR description as well. 
Why are there two files products.js with similar code? Please take a look and sort the files out accordingly. 
Thanks.

sarmadH97 reviewed (commented):
Resolve the added comments.

sarmadH97 commented on frontend/src/pages/products/products.js:
Duplicate file.

augustine24 replied:
I've fixed the issue and I'll attach a picture of the front end with the PR request

augustine24 committed:
removed duplicate files

removed duplicate files, as per request of senior dev sarmad

augustine24 committed:
removed duplicate file

i hadnt noticed there was a duplicate for product.js of my previous work so ive now removed it
